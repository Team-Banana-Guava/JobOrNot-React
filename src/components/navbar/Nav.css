/*

#6290c3 Silver Lake Blue
#62e7da Columbia Blue
#1a1b41 Yankees Blue
#f7fff4 Mint Cream
#baff29 Green-Yellow

*/


/* Navigation Bar & Footer
–––––––––––––––––––––––––––––––––––––––––––––––––– */
nav {
  width: 100%;
  color: #1a1b41;
  background-color: #baff29;
  position: fixed;
  padding: 10px;
  display: inline-block;
}

.navbar a {
  color: #1a1b41;
  text-decoration: none;
}

 .navbar a:hover {
  color: #1a1b41;
  text-decoration: none;
  text-transform: uppercase;
}

.u-pull-right {
  margin-left: 80px;
}
