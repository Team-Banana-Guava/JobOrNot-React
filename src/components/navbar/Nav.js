import React, { PropTypes } from 'react';
import { connect } from 'react-redux';
import SignedInNav from './SignedInNav';
import SignedOutNav from './SignedOutNav';

const Nav = (props) => {
    if (props.isLoggedIn) return (<SignedInNav />);
    else if (!props.isLoggedIn) return (<SignedOutNav />);
};

function mapStateToProps(state) {
    return {
        isLoggedIn: state.userAuth.isLoggedIn
    };
}

export default connect(mapStateToProps)(Nav);

Nav.propTypes = {
    isLoggedIn: PropTypes.bool.isRequired,
};
