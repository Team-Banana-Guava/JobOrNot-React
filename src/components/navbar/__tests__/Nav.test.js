/* eslint-disable */
import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Nav from '../Nav';

describe('Nav', function () {
    it('runs the Nav', () => {
        const rendered = shallow(<Nav signedIn={true} />);
        expect(toJson(rendered)).toMatchSnapshot();
    });
});
