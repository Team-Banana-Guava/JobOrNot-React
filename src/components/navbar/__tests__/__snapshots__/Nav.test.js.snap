exports[`Nav runs the Nav 1`] = `<SignedInNav />`;
